Dockerfile

```diff
@@ -33,6 +33,7 @@ ENV DEBIAN_FRONTEND=noninteractive \
     TAIGA_REDIRECT_TO_SSL=false \
     TAIGA_HOSTNAME=localhost \
     TAIGA_ENABLE_SSL=false \
+    TAIGA_LDAP=false \
```

Comment:
I'm not sure if you want to add all of the new environment variables to the Dockerfile or not, but they all have sensible defaults in the `docker-settings.py` file, so it will work either way.

Reply:
I think that we should put everything we support into the Dockerfile for clarity.
Please add it in format:

```
    # what this does 
    SOME_ENV=defaultvalue \
    # what this does, a comment
    SOME_OTHER_ENV=some
```

As I will port a dynamic documentation generation based on Dockerfile content like it is there: https://github.com/riotkit-org/smtp-ext-relay/blob/master/README.md.j2#L54

Reply:
Reminder: @zicklag I'm waiting for the env change to be applied, then I will merge the PR.